roc/receiver_test.go

```diff
@@ -35,6 +35,46 @@ func TestReceiver_Open(t *testing.T) {
 			config:  ReceiverConfig{},
 			wantErr: newNativeErr("roc_receiver_open()", -1),
 		},
+		{
+			name: "invalid config.TargetLatency",
+			contextFunc: func() *Context {
+				ctx, err := OpenContext(makeContextConfig())
+				require.NoError(t, err)
+				return ctx
+			},
+			config:  makeReceiverConfig(WithTargetLatency(-1)),
+			wantErr: makeTDError(-1, "config.TargetLatency"),
```

Comment:
I think a small portion of copy-paste won't harm in this case and we can avoid introducing extra helper (makeTDError) to keep test table more descriptive:

```
wantErr: fmt.Errorf("invalid config.TargetLatency: %w",
		fmt.Errorf("unexpected negative duration: -1")
```

Like we do in all other tests.

Reply:
no problem
after fix i'll update you via re-request review